test/integration/move.js

```diff
@@ -366,6 +366,50 @@ describe('Move', () => {
           ]
         })
       })
+
+      it('on initial scan with parent move', async () => {
```

Comment:
weird test description :wink:

Reply:
What's the weird part? I would prefer to have the description follow an `it` word but it's not the way other test descriptions have been written in this file :/